ctapipe_io_magic/__init__.py

```diff
@@ -80,6 +82,78 @@
     DATA_MAGIC_LST_TRIGGER: EventType.SUBARRAY,
 }
 
+class ReportLaserContainer(Container):
+    """ Container for Magic laser parameters """
+    UniqueID = Field(None, 'No.')
+    Bits = Field(None, 'ID')
+    MJD = Field(None, 'Modified Julian Date')
+    MiliSec = Field(None, 'Mili sec')
+    NanoSec = Field(None, 'Nano sec')
+    BadReport  = Field(np.bool, 'Bad Report')
+    Telescope = Field(None, 'Telescope')
+    State = Field(None, 'State')
+    IsOffsetCorrection = Field(np.bool, 'Is Offset Correction')
```

Comment:
this np.bool was caught by the automatic tests and they suggested to change it to bool, since np.bool is deprecated.